client/app/bundles/course/assessment/submission/translations.ts

```diff
@@ -260,6 +260,12 @@ const translations = defineMessages({
       Submissions to this assessment will be auto-graded. \
       NOTE THAT THIS ACTION IS IRREVERSIBLE!',
   },
+  fetchSubmissionsFromKoditsuConfirmation: {
+    id: 'course.assessment.submission.fetchSubmissionsFromKoditsuConfirmation',
+    defaultMessage:
+      'Are you sure you want to fetch all submissions from Koditsu? \
+      NOTE THAT THIS ACTION IS IRREVERSIBLE!',
```

Comment:
Need to explain what else will happen, e.g. will it overwrite old answers with newer answers? Will it add newer answers to a submission if there are newer answers in Koditsu?

Reply:
done